File: data/c#/pattern_28/code.cs
using System;
class Grammind
{
    static void Main()
    {
        char ch;
        Console.Write("Enter the number of rows and columns for the square: ");
        ch = Char.Parse(Console.ReadLine());

        for (char i = Char.ToUpper(ch); i >= 'A'; i++)
        {
            for (char j = Char.ToUpper(ch); j >= 'A'; j++)
            {
                Console.Write("{0,2:D }", i);
            }
            Console.Write("\n");
        }
    }
}

File: data/c/pattern_28/code.c
#include <stdio.h>

int main(void)
{
    char ch;
    printf("Enter the character of rows and columns for the square: ");
    scanf("%c", &ch);

    for (char i = toupper(ch); i >= 'A'; i--)
    {
        for (char j = toupper(ch); j >= 'A'; j--)
        {
            printf("%c ", i);
        }
        printf("\n");
    }

    return 0;
}
